README.md

```diff
@@ -113,6 +116,21 @@ Write a conditional statement that...
 ```js
 // your answer here
 ```
+let number =0;
```

Comment:
one way to answer this question

`let a = 5
if (a % 5 == 0 ) {
    console.log("Buzz");
} else if ( a % 3 == 0) {
    console.log("Fizz"); 
} else {
    console.log("FizzBuzz"); 
}`